Regression/WarpX-tests.ini

```diff
@@ -2400,16 +2400,15 @@ dim = 2
 addToCompileString = USE_RZ=TRUE USE_PSATD=TRUE
 cmakeSetupOpts = -DWarpX_DIMS=RZ -DWarpX_PSATD=ON
 restartTest = 0
-useMPI = 1
-numprocs = 2
-useOMP = 1
+useMPI = 0
+numprocs = 1
+useOMP = 0
 numthreads = 1
 compileTest = 0
 doVis = 0
 compareParticles = 1
 particleTypes = driver plasma_e plasma_p
-# TODO: intentionally empty because still unstable
-analysisRoutine =
+analysisRoutine = Examples/analysis_default_regression.py
```

Comment:
needs a custom analysis, see comments and regular `PSATD + RZ` tests in, e.g., PML